media-libs/libheif/libheif-1.17.6.ebuild

```
@@ -0,0 +1,108 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg multilib-minimal
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/strukturag/libheif.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/strukturag/libheif/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~x86"
+fi
+
+DESCRIPTION="ISO/IEC 23008-12:2017 HEIF file format decoder and encoder"
+HOMEPAGE="https://github.com/strukturag/libheif"
+
+LICENSE="GPL-3"
+SLOT="0/1.12"
+IUSE="+aom gdk-pixbuf go rav1e test +threads x265"
+REQUIRED_USE="test? ( go )"
+RESTRICT="!test? ( test )"
+
+# Bug 865351: tests requires <dev-cpp/catch-3
+BDEPEND="
+	test? (
+		<dev-cpp/catch-3
```

https://bugs.gentoo.org/865351 is RESOLVED_OBSOLETE, so drop 
```suggestion
		dev-cpp/catch
```

---

Done in https://github.com/gentoo/gentoo/commit/d3891b55583400ed0aae470559cf9310ebeaf7fd